diff --git a/node_modules/react-native-kakao-share-link/android/src/main/java/com/reactnativekakaosharelink/KakaoShareLinkModule.kt b/node_modules/react-native-kakao-share-link/android/src/main/java/com/reactnativekakaosharelink/KakaoShareLinkModule.kt
index 01c264e..a65e812 100644
--- a/node_modules/react-native-kakao-share-link/android/src/main/java/com/reactnativekakaosharelink/KakaoShareLinkModule.kt
+++ b/node_modules/react-native-kakao-share-link/android/src/main/java/com/reactnativekakaosharelink/KakaoShareLinkModule.kt
@@ -126,11 +126,11 @@ class KakaoShareLinkModule(private val reactContext: ReactApplicationContext) :
 
       // 1. CustomTabs으로 Chrome 브라우저 열기
       try {
-        KakaoCustomTabsClient.openWithDefault(reactContext, sharerUrl)
+        KakaoCustomTabsClient.openWithDefault(reactContext.currentActivity!!, sharerUrl)
       } catch (e: UnsupportedOperationException) {
         // 2. CustomTabs으로 디바이스 기본 브라우저 열기
         try {
-          KakaoCustomTabsClient.open(reactContext, sharerUrl)
+          KakaoCustomTabsClient.open(reactContext.currentActivity!!, sharerUrl)
         } catch (e: ActivityNotFoundException) {
           // 인터넷 브라우저가 없을 때 예외처리
           promise.reject("E_KAKAO_NO_BROWSER", e.message, e)
